site/content/docs/5.0/content/tables.md

```diff
@@ -290,7 +290,7 @@ Table cells of `<thead>` are always vertical aligned to the bottom. Table cells
 </div>
 
 ```html
-<table class="table table-sm table-dark">
+<table class="table table-sm">
```

Comment:
Looking at this HTML, I’m wondering if we should be wrapping everything in this outer table anyway? Normally it’s the `.table-responsive` and then the `.table` within, not an outer one. Can you verify?

Reply:
Good point. In the sample table there is no outer `table` at all.
````HTML
<div class="bd-example">
  <div class="table-responsive">
    <table class="table align-middle">
      <thead>
````
So the hole outer `table` would be removed, right?